helm/values.yaml

```diff
@@ -1,24 +1,104 @@
+    ## https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/user-guide.md
     driver:
-      # Refer to the configuration options available at https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/user-guide.md
       cores: 1
       coreLimit: "1200m"
       memory: "512m"
+
+    ## Use any of the executor configuration options available at
+    ## https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/user-guide.md
     executor:
-      # Refer to the configuration options available at https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/user-guide.md
       cores: 1
       instances: 2
       memory: "512m"
 
+## This section deals with the ingestion components of SDAP
+ingestion:
+
+  enabled: true
+
+  granuleIngester:
+    replicas: 2
+    image: nexusjpl/granule-ingester:0.0.1
+
+    ## cpu refers to both request and limit
+    cpu: 1
+
+    ## memory refers to both request and limit
+    memory: 1Gi
+
+  collectionManager:
+    image: nexusjpl/collection-manager:0.0.2
+
+    ## cpu refers to both request and limit
+    cpu: 0.5
+
+    ## memory refers to both request and limit
+    memory: 0.5Gi
+
+  configOperator:
+    image: nexusjpl/config-operator:0.0.1
+
+  ## How to mount the granule files to ingest
+  granules:
+    ## Enable nfsServer if you want to mount the granules from an NFS server,
+    ## otherwise they will be loaded from the local filesystem.
+    ## mountPath and path should be set whether or not nfsServer is enabled.
+    # nfsServer: nfs-server.com
+
+    ## mountPath is the path in the Collection Manager and Granule Ingester pods
+    ## where the granule files will be mounted.
+    ## IMPORTANT: the `path` property on all collections in the Collections Config file
+    ## should match this value.
+    ## Example: if mountPath is set to /data, then every collection in the Collections
+    ## Config file should have something like
+    ## path: /data/<some-directory>/<some-file-pattern>
+    mountPath: /data
+
+    ## path is the path on either local filesystem or NFS mount at which
+    ## the granule files are stored.
+    path: /var/lib/sdap/granules
+
+  ## Where to find the Collections Config file
+  ## ref: https://github.com/apache/incubator-sdap-ingester/tree/dev/collection_manager#the-collections-configuration-file
+  ## Either localDir should be set, or the git options, but not both.
+  collections:
+
+    ## Load the Collections Config file from a local path
+    ## This is a future option that is not yet supported!
+    # localDir: /Users/edford/Desktop/collections.yml
+
+    ## Load the Collections Config file from a git repository
+    ## Until localDir is supported, this configuration is mandatory
+    git:
+
+
+      ## This should be an https repository url of the form https://github.com/username/repo.git
+      url:
+
+      branch: master
+
+      ## token is not yet supported!
```

Comment:
I guess if we don't have token supported yet, podaac team will not be happy with a public git repository.

When I'll deploy on podaac dev I'll start with local file deployment. That might be what they prefer anyway.

Reply:
Yeah, it looks like we still need to add token support into the config operator code